Libraries/Components/View/ReactNativeViewViewConfigMacOS.js

```diff
@@ -16,22 +16,22 @@ const ReactNativeViewViewConfigMacOS = {
   uiViewClassName: 'RCTView',
   directEventTypes: {
     topDoubleClick: {
```

Comment:
For my education, why should this stay as topDoubleClick (and all the other ones stay as top*? Should they not switch to on* to match?

Reply:
As I understand it its a react.js thing.   The 'onFoo' events are the names of the actually callbacks to app code.   The 'topFoo' is some kind of convention for the origin of bubbling events.   All the cases of bubbling events follow this convention and we (I?) made the mistake of misnaming the registrationName for all these events when creating this file recently.

Reply:
If i had a guess as to who mistakenly changed it, you would be a distant 2nd on the list :)

Reply:
Yeah, the `top` keys is a whole thing that is legacy. It's not easy to change those without breaking things though, so it will probably stick around for the forseeable future. Although we *all* want to change it. :D